### DISCUSSION TITLE
[Feature Request] BiomeFixAll: Inspired by EslintFixAll

### DISCUSSION THREAD

**Discussion by jellydn** [biomejs/biome#2527] [Ideas]
# What

BiomeFixAll is a proposed tool designed to automatically fix common and repetitive coding issues within the Biome development environment. This tool would operate similarly to [EslintFixAll](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/linting/eslint.lua#L40) command.

# Why

The introduction of BiomeFixAll would significantly improve developer productivity and code quality by:

 - Reducing manual effort: Automatically fixing routine issues frees up developers to focus on more complex and creative problem-solving.
 - Ensuring consistency: Applying uniform standards across the codebase enhances readability and maintainability.
 - Speeding up the development cycle: Quick fixes for common problems can accelerate development processes and reduce the time to deployment.

# How

TBD

**Comment by hilja** [comment]
How does this differ from `biome check --apply-unsafe .`?

**Reply by jellydn** [reply]
Hi @hilja 
You're right. That is my workaround right now.

However, I am looking for a LSP solution like this how ESlint works with my nvim [config](https://github.com/jellydn/lazy-nvim-ide/commit/ea149f11342aabf3ff2ab974a5ae80ecd0d522f0) 
[![Image from Gyazo](https://i.gyazo.com/e30020ffcc22809171679cd3ce27b73f.gif)](https://gyazo.com/e30020ffcc22809171679cd3ce27b73f)
